.github/workflows/ci-publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish to NPM
+on: 
+  push:
+    tags:
+      - '*'
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: install
+      - uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: test
+
+  publish-npm:
+    needs: build
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: borales/actions-yarn@v2.0.0
```

Comment:
use `name: ` in all `borales/actions-yarn@v2.0.0` steps, otherwise they are just named `Run borales/actions-yarn@v2.0.0` in [the output](https://github.com/libero/eslint-config/runs/410522936?check_suite_focus=true)

Reply:
done